.travis.yml

```diff
@@ -1,4 +1,4 @@
-# Created with package:mono_repo v2.1.0
+# Created with package:mono_repo v2.0.0
```

Comment:
upgrade to latest mono_repo – is nice! @jakemac53

Reply:
hmm it seems like mono_repo should warn about this haha (running on an older version than the last run was done with?)

Reply:
https://github.com/dart-lang/mono_repo/issues/146 @nshahan is ahead of you!